mobile/lib/modules/explore/widgets/explore_item.widget.dart

```diff
@@ -55,7 +55,7 @@ class ExploreItem extends StatelessWidget {
                         borderRadius: BorderRadius.all(Radius.circular(8)),
                       ),
                       child: Text(
-                        item.organizationName,
+                        item.organizationName!,
```

Comment:
I recommend using null check and add default value for null safety `currentCampaign.organizationName ?? 'Some foo default value'`